### PR TITLE
[CVE-2022-48285][1.x] Bump jszip from 3.7.1 to 3.10.1

### PR DESCRIPTION
**Pull request by ananzh**
### Description
This CVE is caught in github. It is about that `loadAsync` in `jszip` before 3.8.0 allows Directory Traversal via a crafted ZIP archive. `jszip` is a dependency of `selenium-webdriver` . This CVE requires to bump `jszip` to 3.8.0+. In 1.x, we are currently using `jszip@3.7.1` .

```
ubuntu@ip-172-31-55-237:~/work/OpenSearch-Dashboards$ yarn why jszip
yarn why v1.22.19
[1/4] Why do we have the module "jszip"...?
[2/4] Initialising dependency graph...
warning Resolution field "typescript@4.0.2" is incompatible with requested version "typescript@~3.7.2"
warning Resolution field "shelljs@0.8.5" is incompatible with requested version "shelljs@^0.6.0"
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "jszip@3.7.1"
info Reasons this module exists
   - "_project_#selenium-webdriver" depends on it
   - Hoisted from "_project_#selenium-webdriver#jszip"
info Disk size without dependencies: "796KB"
info Disk size with unique dependencies: "1.93MB"
info Disk size with transitive dependencies: "2.21MB"
info Number of shared dependencies: 11
Done in 1.05s.
```
```
ubuntu@ip-172-31-55-237:~/work/OpenSearch-Dashboards$ yarn why selenium-webdriver
yarn why v1.22.19
[1/4] Why do we have the module "selenium-webdriver"...?
[2/4] Initialising dependency graph...
warning Resolution field "typescript@4.0.2" is incompatible with requested version "typescript@~3.7.2"
warning Resolution field "shelljs@0.8.5" is incompatible with requested version "shelljs@^0.6.0"
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "selenium-webdriver@4.0.0-alpha.7"
info Has been hoisted to "selenium-webdriver"
info Reasons this module exists
   - "workspace-aggregator-b86b8a47-7a5d-4547-95f7-e0849c232835" depends on it
   - Specified in "devDependencies"
   - Hoisted from "_project_#selenium-webdriver"
info Disk size without dependencies: "972KB"
info Disk size with unique dependencies: "1.79MB"
info Disk size with transitive dependencies: "3.52MB"
info Number of shared dependencies: 24
Done in 1.04s.
```
`selenium-webdriver` doesn’t actively upgrade `jszip` in [package.json](https://github.com/SeleniumHQ/selenium/blob/trunk/package.json), which is still `"^3.7.1"` and from [4.7.0](https://github.com/SeleniumHQ/selenium/blob/selenium-4.7.0/package-lock.json) they actually start to use `3.10.1` (package-lock). Since we are gradually remove all selenium tests. I think it is a bit waste of time to investigate the changes between "selenium-webdriver@4.0.0-alpha.7" and 4.7.0. 

Since there is [no breaking changes](https://github.com/Stuk/jszip/commits/v3.8.0) between 3.7.1 and 3.8.0 on `jszip`, We could just make a simple resolution.
 
 
### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,6 +102,7 @@
     "**/jest-jasmine2": "npm:@amoo-miki/jest-jasmine2@26.4.2-rc.1",
     "**/jsonpointer": "^5.0.0",
     "**/json-schema": "^0.4.0",
+    "**/jszip": "^3.8.0",
```
Comment (AMoo-Miki) [inline package.json:105]: This shouldn't be needed.

**Comment by joshuarrrr** [comment]
@ananzh Marking as draft until you have time to revisit Miki's comments.

File: package.json (hunk 1 above)
Comment (ananzh) [reply package.json:105]: Yeah it is true. If we bootstrap, we will get an update.

**Review by joshuarrrr — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (joshuarrrr) [reply package.json]: I removed the resolution and did a clean bootstrap as recommended.

**Review by manasvinibs — APPROVED** [review]

**Review by seanneumann — APPROVED** [review]
👍

**Pull request merged**